Pennsieve/pennsieve-app
# Create account on login - CU-tq5jj5

# Description

The purpose of this PR is to add the create account workflow for a user.

## Clickup Ticket

[tq5jj5](https://app.clickup.com/t/tq5jj5)

## Type of change


# How Has This Been Tested?

This has CSP changes, so you will need to restart your dev server.

- On the login screen, click on "Create one here"
- You should taken to the create account screen
- The reCAPTCHA badge should be in the bottom left
- All fields should be required
- Enter all fields
- Click on "Create Account"
- You should get an error message (due to  the backend not being done yet)
- Clicking on "contact support" in the error message should open Intercom
- Using vue devtools, inspect `CreateAccount` and set `accountCreated` to `true`
- You should see the success message
- Click on the "Back to login" page
- You should be  taken back to the login screen
- The reCAPTCHA badge should be gone

To test success, I used the following mock URL in place of the `/sign-up` endpoint https://run.mocky.io/v3/14d22f0f-6ea7-4e96-bc8e-362eb8752945

# Checklist:
